.github/workflows/ci.yml

```
@@ -287,7 +287,7 @@ jobs:
           crate: cargo-toml-lint
           version: '0.1'
       - name: Run Cargo.toml linter
-        run: git ls-files | grep Cargo.toml$ | grep -v 'templates/sway-test-rs' | xargs --verbose -n 1 cargo-toml-lint
+        run: git ls-files | grep Cargo.toml$ | grep -v 'templates/sway-test-rs' | grep -v 'templates/sway-predicate-test-rs' | grep -v 'templates/sway-script-test-rs' | xargs --verbose -n 1 cargo-toml-lint
```

Linting fails because of the template characters `{{` and `}}`. We can skip linting for all Cargo.toml files in this directory.

```suggestion
        run: git ls-files | grep Cargo.toml$ | grep -v 'templates/' | xargs --verbose -n 1 cargo-toml-lint
```